src/main/java/appeng/client/render/BaseBlockRender.java

```diff
@@ -321,17 +321,23 @@ public static int getOrientation( ForgeDirection in, ForgeDirection forward, For
 			return 0;
 		}
 
-		int a = in.ordinal();
-		int b = forward.ordinal();
-		int c = up.ordinal();
+		final int a = in.ordinal();
+		final int b = forward.ordinal();
+		final int c = up.ordinal();
 
 		return ORIENTATION_MAP[a][b][c];
 	}
 
 	public void renderInvBlock( EnumSet<ForgeDirection> sides, B block, ItemStack item, Tessellator tess, int color, RenderBlocks renderer )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "renderInvBlock" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)